pkg/runtime/runtime.go

```diff
@@ -1704,6 +1700,12 @@ func (a *DaprRuntime) startReadingFromBindings() error {
 	}
 	for name, binding := range a.inputBindings {
 		go func(name string, binding bindings.InputBinding) {
+			subscribed := a.isAppSubscribedToBinding(name)
+			if !subscribed {
+				log.Infof("app-id=%s has not subscribed to binding %s.", a.runtimeConfig.ID, name)
```

Comment:
Since this log is per Dapr deployment (app-id), I think we can remove the `app-id` portion here. this is consistent with other logs in `daprd`